src/main/java/ch/njol/skript/expressions/ExprExperience.java

```diff
@@ -48,37 +46,41 @@
 		"\tclear dropped experience",
 		"on break of diamond ore:",
 		"\tif tool of player = diamond pickaxe:",
-		"\t\tadd 100 to dropped experience"})
+		"\t\tadd 100 to dropped experience",
+		"on fishing:",
+		"\tadd 70 to dropped experience"})
 @Since("2.1, 2.5.3 (block break event)")
-@Events({"experience spawn", "break / mine"})
+@Events({"experience spawn", "break / mine", "fish"})
```

Comment:
I think the event is technically `fishing`

Reply:
Glad to help :pray:

> I think the event is technically fishing

True, I thought the event was `fish` because this is linked with the documentation but it seems like it is `fishing` indeed, good catch ;)